### DISCUSSION TITLE
Unable to finish generating the text

### DISCUSSION THREAD

**Discussion by iLarryG** [GorvGoyl/ChatGPT-Writer#15] [Report Bugs and Issues]
[The extension starts generating the response and then suddenly stops for no reason](https://drive.google.com/file/d/1NLwbgsNPa7UcGlcvLh5RTWG8Yk2VoD-a/view?usp=sharing)

**Comment by alexkuttig** [comment]
+1

**Comment by GorvGoyl** [comment]
I fixed it.. please wait for the upcoming release v2.0.. should be live by tomorrow

**Reply by sanjaysingh08** [reply]
Unbale to generate text

**Comment by GorvGoyl** [comment]
v2.0 is live on the chrome store. please update and let me know if you are still facing the issue

**Comment by iLarryG** [comment]
It works now. It's really amazing. Thank you ! 
(even though the button "insert generated response" doesn't work, we can always copy/paste ;) )

**Comment by anewbies** [comment]
Se estiver usando o Google Chrome, entre na configuração chrome://settings/syncSetup e marque a caixa on "verificação de ortografia aprimorada" e o problema estará resolvido.
![image](https://user-images.githubusercontent.com/14927641/213434388-b31201f0-4ac8-4fcc-a0eb-626e83ff2ef3.png)
